### PR TITLE
🎉 Source Chargebee: add event model

### PR DESCRIPTION
**Pull request by joelluijmes**
## What
This PR adds [Events](https://apidocs.eu.chargebee.com/docs/api/events?prod_cat_ver=2#list_events) model to the Chargebee connector.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### CODE REVIEW

File: airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json [hunk 1]
```diff
@@ -103,6 +103,19 @@
...
+        "name": "event",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": true,
+        "default_cursor_field": ["occurred_at"],
+        "source_defined_primary_key": [["id"]]
+      },
+      "sync_mode": "incremental",
+      "destination_sync_mode": "append",
+      "cursor_field": ["occurred_at"]
```
Comment (joelluijmes) [inline airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json:118]: I've no idea if this is necessary. Is this part used during integration test? Or is it just for documentation purposes? I guess we'll see when tests run 🤓

Comment (joelluijmes) [inline airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json:118]: I've no idea if this is necessary. Is this part used during integration test? Or is it just for documentation purposes? I guess we'll see when tests run 🤓

Comment (joelluijmes) [inline airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json:118]: I've no idea if this is necessary. Is this part used during integration test? Or is it just for documentation purposes? I guess we'll see when tests run 🤓

**Comment by harshithmullapudi** [comment]
Thanks for the contribution. Team will look into this soon

File: docs/integrations/sources/chargebee.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ Log into Chargebee and then generate an [API Key](https://apidocs.chargebee.com/
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.6 | XXX | [XXX](https://github.com/airbytehq/airbyte/pull/XXX) | Add support for Event stream |
```
Comment (marcosmarxm) [inline docs/integrations/sources/chargebee.md:84]: can you update here? @joelluijmes

File: airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json (hunk 1 above)
Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json:118]: It's used for integration tests

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
looks great @joelluijmes I'll test it tomorrow.

File: airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json (hunk 1 above)
Comment (joelluijmes) [reply airbyte-integrations/connectors/source-chargebee/integration_tests/configured_catalog.json:118]: Ah okay, didn't got integrations tests to run locally. Bunch of authentication errors (not sure if it expects to talk to actual chargebee?).
Regardless, I should probably also edit the other files there then. Any clues? Or should I wait until the CI is triggered.

File: docs/integrations/sources/chargebee.md (hunk 2 above)
Comment (joelluijmes) [reply docs/integrations/sources/chargebee.md:84]: Sure, what would you have on the date? For now I'll add tomorrow (if you have then time to test it), but wondering in general. or should it be done when its actually merged?

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**